app/domain/authentication/authn_jwt/input_validation/validate_claim_name.rb

```diff
@@ -0,0 +1,55 @@
+module Authentication
+  module AuthnJwt
+    module InputValidation
+      # Validate the claim name value
+      ValidateClaimName ||= CommandClass.new(
+        dependencies: {
+          regexp_class: Regexp,
+          deny_claims_list_value: [],
+          logger: Rails.logger
+        },
+        inputs: %i[claim_name]
+      ) do
+        def call
+          @logger.debug(LogMessages::Authentication::AuthnJwt::ValidatingClaimName.new(@claim_name))
+          validate_claim_name_exists
+          validate_claim_name_value
+          validate_claim_is_allowed
+          @logger.debug(LogMessages::Authentication::AuthnJwt::ValidatedClaimName.new(@claim_name))
+        end
+
+        private
+
+        def validate_claim_name_exists
+          raise Errors::Authentication::AuthnJwt::FailedToValidateClaimMissingClaimName if @claim_name.blank?
```

Comment:
i think that `MissingClaimName` is enough and much more readable

Reply:
also - i think that `EmptyClaimName` is better

Reply:
I think it is to general name and we may need it in the future in other flow